src/api/app/controllers/person/token_controller.rb

```diff
@@ -36,6 +36,30 @@ def delete
       render_ok
     end
 
+    # PUT /person/<login>/token/<id>
+    def update
+      authorize @user, :update?
+
+      permitted_params = params.slice(:enabled).permit(:enabled)
```

Comment:
Uhm params? We should read XML no?

Reply:
Uhm okay...

https://trello.com/c/7Acu3hrm

Reply:
If you are fine, I can introduce the PUT endpoint requiring an XML as a request body. I modified the PR to using query parameters just to align with the POST endpoint, after I realized how the POST endpoint was developed.

Reply:
I don't mind anything except merging more code that we have to change later on.

Reply:
Now, the PUT endpoint reads XML. Please take a look again.